python/dgl/nn/pytorch/conv/appnpconv.py

```diff
@@ -99,8 +102,10 @@ def forward(self, graph, feat):
                 # normalization by src node
```

Comment:
Should edge weight also participate in symmetric adjacency normalization as in [this implementation](https://pytorch-geometric.readthedocs.io/en/latest/modules/nn.html#torch_geometric.nn.conv.APPNP)?